cou/utils/juju_utils.py

```diff
@@ -139,16 +139,9 @@ class COUMachine:
 
     machine_id: str
     hostname: str
+    apps: tuple[tuple[str, str], ...]
```

Comment:
This is not hosting just `apps` so I think we should call something more meaningful like `apps_charms` or `apps_charm_names`